aldryn_events/tests/frontend/integration/pages/page.add.delete.event.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global by, element, browser, expect */
+
+// #############################################################################
+// INTEGRATION TEST PAGE OBJECT
+
+var aldrynEventsPage = {
+    site: 'http://127.0.0.1:8000/en/',
+    mainElementsWaitTime: 12000,
+
+    // log in, log out
+    editModeLink: element(by.css('.inner a[href="/?edit"]')),
+    loginUsername: 'admin',
+    loginPassword: 'admin',
+    usernameInput: element(by.id('id_cms-username')),
+    passwordInput: element(by.id('id_cms-password')),
+    loginButton: element(by.css('.cms_form-login input[type="submit"]')),
+    userMenuOptions: element.all(by.css('.cms_toolbar-item-navigation li')),
+
+    loginToSite: function (emailInput, email, passwordInput, password,
```

Comment:
it's better to do:

```
loginToSite: function (object) ...
```

and than call

```
loginToSite({
    email: 'blahblah',
    password: 'blahblah'
});
```

you can hardcode the login input and password input as they should always be the same on django-cms